client/src/components/Profile/SidebarStyles.js

```diff
@@ -0,0 +1,136 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles(theme => ({
+    linkStyles: {
+        textDecoration: 'none',
+        color: 'black',
+        padding: '1em 0 0 2em',
+        fontSize: 18,
+        opacity: '50%',
+    },
+    profilePic: {
+        borderRadius: '50%',
+        width: '130px',
+        marginTop: '4em',
+    },
+    drawer: {
+        background: '#F9F9FC',
+    },
+    linkContainer: {
+        paddingTop: '2em',
+    },
+    profileName: {
+        paddingTop: '1em',
+        fontSize: 22,
+    },
+    list_container: {
+        display: 'block',
+        height: 'calc(100vh - 103px)',
+        overflowY: 'auto',
+        '&::-webkit-scrollbar': {
+            width: '0.6rem',
+            height: '12rem',
+        },
+        '&::-webkit-scrollbar-track': {
+            boxShadow: 'inset 0 0 5px grey',
+            borderRadius: 4,
+        },
+        '&::-webkit-scrollbar-thumb': {
+            backgroundColor: 'rgba(0,0,0,.1)',
+            borderRadius: 4,
+        },
+        '&::-webkit-scrollbar-thumb:hover': {
+            background: '#ccc',
+        },
+    },
+    chat_head: {
+        height: 120,
+        display: 'flex',
+        justifyContent: 'flex-start',
+        alignItems: 'center',
+        paddingLeft: theme.spacing(4),
+        '& p': {
+            fontSize: 22,
```

Comment:
I don't recall what the standard has been so far, but you may want to use rem for font sizes instead of pixels.

Reply:
Changed all font sizes to rem.